packages/nodejs/project.bri

```diff
@@ -5,13 +5,13 @@ import nushell, { nushellRunnable, type NushellRunnable } from "nushell";
 
 export const project = {
   name: "nodejs",
-  version: "24.4.0",
+  version: "24.4.1",
   extra: {
     currentMajorVersion: "24",
     majorVersions: {
-      "24": "24.4.0",
-      "22": "22.17.0",
-      "20": "20.19.3",
+      "24": "24.4.1",
+      "22": "22.17.1",
+      "20": "20.19.4",
     },
   },
 } as const;
```

Comment:
That's understandable, it is not serializable when we call `JSON.stringify(project)`

Reply:
Yeah, this looks like a bug in Brioche... just opened a follow-up issue here: https://github.com/brioche-dev/brioche/issues/283